requirements/requirements.in

```diff
@@ -8,7 +8,7 @@ filelock
 Flask
 Flask-Cors
 Flask-RESTful
-gevent==1.5a3
+gevent>=1.5a3
```

Comment:
why not keep gevent pinned?

Reply:
The recommended way to use the .in files is to pin as little as possible there and instead selectively upgrade the compiled result.